lib/iris/io/__init__.py

```diff
@@ -169,11 +169,10 @@ def expand_filespecs(file_specs):
         msg = "One or more of the files specified did not exist:"
         for pattern, expanded in six.iteritems(glob_expanded):
             if expanded:
-                file_list = '\n       - {}'.format(', '.join(expanded))
+                msg += '\n    - "{}" matched {} file(s)'.format(pattern,
+                                                                len(expanded))
             else:
-                file_list = ''
-            msg += '\n    - "{}" matched {} file(s){}'.format(
-                    pattern, len(expanded), file_list)
+                msg += '\n    * "{}" expanded to empty'.format(pattern)
```

Comment:
Trying to think how this would look to a newbie - what expanded to empty?

Ideally, I would like some variant of "didn't match any files".